dev/com.ibm.ws.logging/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -45,15 +45,15 @@ console.level.info=Info, audit, and warning messages will be written to the syst
 console.level.audit=Audit and warning messages will be written to the system output stream. Error messages will be written to the system error stream.
 console.level.warning=Warning messages will be written to the system output stream. Error messages will be written to the system error stream.
 console.level.error=Error messages will be written to the system error stream.
-console.level.off=No server output will be written to system streams.� Only JVM output will be written to system streams.
+console.level.off=No server output will be written to system streams. Only JVM output will be written to system streams.
```

Comment:
Suggestion for line 48: Server output will not be written to system streams. Only JVM output will be written to system streams.